.eslintrc.json

```diff
@@ -1,10 +1,8 @@
 {
   "extends": ["plugin:react/recommended", "@appium/eslint-config-appium"],
```

Comment:
would it be possible to use eslint-config-appium-ts instead of eslint-config-appium later and have .ts files supported?

Reply:
Changed to the TS module. In order to pass linting now, I had to change `electron-debug` and `electron-extension-installer` `require()`s to top-level `import`s (and therefore also change them from dev deps to main deps), but this is also how their README suggests them to be used.